Changes to perform joint dfs0 and res11 read

LAGO-Support opened the pull request:
These are the changes to address the comments made by Kiren, as well as test the command line push.

LAGO-Support committed:
Changes to perform joint dfs0 and res11 read

kirenbahm commented:
Thanks for the commit. Overall things look good, except it isn't quite working for me. 
This is the message I am getting:
```
WARNING: missing M11 file MSHE_WM for:.dfs0
WARNING: missing M11 file MSHE_WM for:.res11
```
 Here are the values of the relevant variables:
```
INI.fileM11WM
  ans =   '..\..\ENP_FILES\ENP_TOOLS_Sample_Input\Result\M01_test_short.she - Result Files\MSHE_WM.res11'
ext =     1     2     4     5    61    88
fe =    '.res11'
fn =    'MSHE_WM'
fp =    '..\..\ENP_FILES\ENP_TOOLS_Sample_Input\Result\M01_test_short.she - Result Files'
dfs0Exists =     0
dfs0File =    '.dfs0'
res11File =    '.res11'
res11Exists =     0
```

Can you assist?

LAGO-Support committed:
FileName Parse Fix

Previous code didn't account for '.' appearing multiple times in the file name.
